sitewit/models.py

```diff
@@ -205,6 +206,10 @@ def _get_valid_user_name(cls, user_name):
     def _get_email(cls, user):
         return user.preferences.get('wl_email', user.email)
 
+    @classmethod
+    def _generate_user_id(cls, user):
```

Comment:
Just thinking... What if we call it `_generate_sw_user_id` just to distinguish it from Yola's `user_id`?

Reply:
makes sense